test/train-sets/ref/metrics_ccb_parse_error_limit.stderr

```diff
@@ -0,0 +1,27 @@
+Num weight bits = 18
+learning rate = 0.5
+initial_t = 0
+power_t = 0.5
+using no cache
+Reading datafile = train-sets/ccb_data_parse_error.json
+WARNING: Old string feature value behavior is deprecated in JSON/DSJSON and will be removed in a future version. Use `--chain_hash` to use new behavior and silence this warning.
+num sources = 1
+Enabled reductions: gd, generate_interactions, scorer, csoaa_ldf, cb_adf, cb_explore_adf_greedy, cb_sample, shared_feature_merger, ccb_explore_adf, extra_metrics
+average  since         example        example  current  current  current
+loss     last          counter         weight    label  predict features
+0.000000 0.000000            1            1.0  3:0,0:0      3,0       48
+-0.300000 -0.600000            2            2.0 1:-0.6,3:-0.4      1,3       48
+-0.162500 -0.025000            4            4.0  2:0,0:0      2,0       48
+-0.093875 -0.025250            8            8.0  2:-0.42        2       35
+-0.533937 -0.974000           16           16.0      1:0        1       35
+-0.328594 -0.123250           32           32.0  1:-0.42        1       35
+[error] JSON parser error at 43: Missing a comma or '}' after an object member.. Handler:  State: DecisionService
+-0.547531 -0.766469           64           64.0      1:0        1       35
+
+finished run
+number of examples = 68
+weighted example sum = 68.000000
+weighted label sum = 0.000000
+average loss = -0.572868
+total feature number = 2471
+[critical] Ommitted some log lines. Re-run without --limit_output N for full log. Total 3
```

Comment:
what's `Total 3`? there we total 3 lines and some were omitted? (Also nit omitted has one `m`)

Reply:
fixed, its the total log lines